tokenlearn/train.py

```diff
@@ -18,7 +18,7 @@
 
 
 def train_model(
-    model_name: str, train_txt: list[str], train_vec: np.ndarray, device: str = "cpu", vocab_size: int | None = None
+    model_name: str, train_txt: list[str], train_vec: np.ndarray, device: str = "cpu",batch_size: int= 32, vocab_size: int | None = None,trust_remote_code: bool=True
```

Comment:
Small styling issue:
```suggestion
    model_name: str, train_txt: list[str], train_vec: np.ndarray, device: str = "cpu", batch_size: int= 32, vocab_size: int | None = None, trust_remote_code: bool=True
```

Reply:
Also on a couple of other lines, see general PR comment.